frontend/src/views/ChannelView.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+	<div class="page_box_wrapper">
+		<div class="page_box">
+			<h1>This is a channel page</h1> //!!! --> not sure is we get a separate channel vieuw page? as in only one button is different so maybe there are other ways, else, here it is!
+		</div>
```

Comment:
this is not a html comment ;-) so this text just shows up on the page
which is not really a problem, since this page is not made, just so you know